frontend/src/components/forms/CreateResident.tsx

```diff
@@ -106,9 +102,22 @@ const CreateResident = ({
     }
   };
 
+  const getBuildingsOptions = async () => {
+    const buildingsData = await BuildingAPIClient.getBuildings();
+
+    if (buildingsData && buildingsData.buildings.length !== 0) {
+      const buildingLabels: BuildingLabel[] = buildingsData.buildings.map(
+        (building) => ({ label: building.name!, value: building.id! }),
```

Comment:
I removed the exclamation marks after the properties and it still works. What do the exclamation marks do?

Reply:
it asserts that the name or id are not null (ngl i copied it from where else in the code where they did the same thing)